src/test/unit/test_config.py

```diff
@@ -1,43 +1,31 @@
 import pydantic
 import pytest
 
+import config
+
 # We explicitly don't want the patch_cfg fixture to be able to patch this function
 # This is why we import it here
 from config import load
-from config import cfg, configparser_cfg, parse_comma_separated_list
 from test.common_helper import get_test_data_dir
 
 
-def test_load():
-    cfg_path = get_test_data_dir() + '/load_cfg_test'
+def test_load(monkeypatch):
+    # Undo all monkeypatching which includes what `patch_config` patched.
+    monkeypatch.undo()
+    cfg_path = get_test_data_dir() + '/fact-core.toml'
```

Comment:
We agreed internally to use f-strings for string formatting
```suggestion
    cfg_path = f'{get_test_data_dir()}/fact-core.toml'
```